gui/Test/src/main/java/sample/test/controllers/LoginViewController.java

```diff
@@ -33,6 +36,8 @@ public class LoginViewController implements Initializable {
     @FXML
     private PasswordField enterPasswordField;
 
+    private String jwtToken;
```

Comment:
I noticed that the current implementation of the JWT token in the LoginViewController has a limitation: the token does not persist across view changes or reloads. When the view is reloaded or the user navigates to another screen, the token will be lost, which can lead to authentication issues.

To address this, I recommend implementing a singleton pattern or a dedicated service to manage the JWT token. This way, we can ensure that the token remains accessible throughout the application lifecycle.

Reply:
Got it. Thanks for the feedback, I will fix it right away.